### DISCUSSION TITLE
Search queries as links

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#6696] [Q&A] [answered]
### Description

Take as an example this blog: <https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/>

When I put, a term in the filter-ui it gives me the results that match the term. For example, when I put `access` it gives 5 posts as a result.

What I want is to have that in form of a link (a search query), so you can see exactly those same results when clicking the link. Something as this https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/search?q=access but that actually work

**Comment by mcanouil** [comment]
The search engine is provided by Algolia (<https://www.algolia.com/doc/>).

It seems to be related to the `InstantSearch.js` module (<https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/>) which is not provided inside Quarto currently.

@dragonstyle Do you think, this feature could be added to the search engine in a future version?

**Reply by dragonstyle** [reply]
Listing filtering is actually implemented slightly differently - using a library called listing.js which provides filtering, pagination, and sorting (not using Algolia which actually currently does supporting linking to searches). This could be supported and would be a nice enhancement.

**Reply by dragonstyle** [reply, marked as answer]
Created #6755 to track.
